app/index.html

```diff
@@ -3,9 +3,11 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>WQXR | New York's Classical Music Station</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="preload" href="{{rootURL}}assets/vendor.js" as="script"
```

Comment:
Does this link tag need a closing ">"?

Reply:
haha yes, ty! that's what I get for doing this late.